azure-pipelines.yml

```diff
@@ -194,3 +194,24 @@ jobs:
 
   - script: python setup.py sdist
     displayName: 'Build sdist'
+
+
+- job: DocsTest
+  pool:
+    vmImage: 'ubuntu-16.04'
+  steps:
+  - bash: echo "##vso[task.prependpath]/usr/share/miniconda/bin"
+    displayName: Add conda to PATH
+  - script: conda env create -n test_env --quiet --file ci/doc.yml
+    displayName: Create Anaconda environment
+  - script: |
+      source activate test_env
+      # Replicate the exact environment created by the readthedocs CI
```

Comment:
do we agree now that this is not the goal?

Reply:
I think it is the goal for the CI isn't it?

Reply:
I think the goal is to test the docs, not debug the service that builds/serves the docs. We test pvlib on many different environments, so it's reasonable to test the docs in different environments too.